tests/ttnn/unit_tests/test_reshape.py

```diff
@@ -285,3 +281,46 @@ def test_reshape_tile_layout_only_change_shape(device):
     output = ttnn.to_torch(ttnn_output)
 
     assert_with_pcc(torch_result, output, 0.9999)
+
+
+# Reshape in Tile layout with shapes that are not divisible by 32
+@pytest.mark.parametrize(
```

Comment:
Thinking of more test cases like these
((1, 1, 1, 16, 2), (1, 1, 1, 32)),
((1, 1, 1, 4, 4), (1, -1, 4)),
((1, 1, 1, 8, 8, 1024), (1, 64, 1024))


from here
https://github.com/tenstorrent/pytorch2.0_ttnn/blob/kw/metrics-support-more-statuses/docs/operations/aten.view.default.md

https://github.com/tenstorrent/pytorch2.0_ttnn/blob/kw/metrics-support-more-statuses/docs/operations/aten._unsafe_view.default.md

Reply:
I think at the moment, we cannot support any of these because the inputs are more than rank 4. 

I can add this support in a subsequent PR.